packages/dialtone-vue2/components/input/input.vue

```diff
@@ -649,3 +585,9 @@ export default {
   },
 };
 </script>
+
+<style lang="less">
+.d-input--hidden {
+  visibility: hidden;
+}
```

Comment:
Is there a reason why this `.d-input--hidden` is not in `input.less` file?

Im guessing we should also update the `input.json` file with this class `apps/dialtone-documentation/docs/_data/input.json`

Reply:
guessing this happened when we changed from using util classes in dialtone. I see no reason this can't be moved to  input.less

Reply:
I added this, forgot to move it to input.less